### PR TITLE
Added main tag to all pages

### PR DESCRIPTION
**Pull request by jadonwolffs**
As per issue #427 I went through all the pages of the site and added <main> tags to wrap the main content of the 2 pages that I found to not have it

I tried to use the fix and lint scripts to keep the formatting consistent but some things did change the first time I did it so I re-forked and added the tags again

### CODE REVIEW

File: src/id-data-controls.js [hunk 1]
```diff
@@ -39,115 +39,117 @@ class IdDataControls extends preact.Component {
...
-                                            </label>
-                                        </strong>
-                                    </div>
-                                    <div className="col60" style="padding-left: 10px; box-sizing: border-box;">
-                                        <InputControl
-                                            id="name-input"
-                                            suffix="fixed-id-data"
-                                            onChange={e => this.handleFixedChange('name', e)}
-                                            value={this.state.fixed_id_data['name']}
+                <main>
```
Comment (zner0L) [inline src/id-data-controls.js:42]: How about adding the main tag just at the end here:

https://github.com/datenanfragen/website/blob/f07be204fdc5663934cdb32b19679f423b7cbbc9/src/id-data-controls.js#L241-L247

**Review by zner0L — CHANGES_REQUESTED** [review]
Thanks for your contribution! Take a look at the comments and then I am happy with merging :)

File: src/id-data-controls.js (hunk 1 above)
Comment (jadonwolffs) [reply src/id-data-controls.js]: That's a much better way of doing that, I've made the changes I think

**Review by zner0L — APPROVED** [review]
Thanks!

**Pull request merged**